pkg/config/config.go

```diff
@@ -197,3 +143,58 @@ func validateConfig(cfg *latest.Config) error {
 func boolPtr(b bool) *bool {
 	return &b
 }
+
+func ValidatePathInDirectory(path, allowedDir string) (string, error) {
```

Comment:
This one doesn't look used any more? Or is it used somewhere else?

Reply:
Yeah, it's used in many other places in the codebase...